cmd/jag/commands/decode.go

```diff
@@ -43,6 +43,7 @@ func DecodeCmd() *cobra.Command {
 	}
 	cmd.Flags().BoolP("force-pretty", "r", false, "force output to use terminal graphics")
 	cmd.Flags().BoolP("force-plain", "l", false, "force output to use plain ASCII text")
+	cmd.Flags().String("envelope", "", "path to the firmware envelope")
```

Comment:
It doesn't really have to be a path.

```suggestion
	cmd.Flags().String("envelope", "", "name or path of the firmware envelope")
```